instrumentation/http-tests/src/main/java/brave/http/ITHttp.java

```diff
@@ -41,6 +43,7 @@
           }
           spans.add(s);
         })
+        .propagationFactory(ExtraFieldPropagation.newFactory(Propagation.Factory.B3, "user-id"))
```

Comment:
so this means do B3 and also if you see "user-id" pass it along and onto outgoing headers